### PR TITLE
Fix font to be the same as design 

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

File: tuneme/templates/_base.html [hunk 1]
```diff
@@ -5,15 +5,18 @@
...
+        <link href='http://fonts.googleapis.com/css?family=Quicksand:400,700' rel='stylesheet' type='text/css'>
         <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}{% block title_suffix %}{% endblock %}</title>
-        <meta name="description" content="" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-
+        <meta http-equiv="content-type" content="text/html;charset=utf-8" />
+        <meta name="description" content="NDOH" />
+        <meta name="keywords" content="NDOH" />
+        <meta property="og:title" content="NDOH" />
+        <meta property="og:description" content="NDOH">
```
Comment (smn) [inline tuneme/templates/_base.html]: I think these `NDOH` bits are probably copy & paste errors?

**Comment by smarzban** [comment]
@smn I wasn't sure about them I saw them in the design, can you explain what those are for? should I remove them or change the `NDOH` to `TUNEME`/or something else?

**Comment by smn** [comment]
@stevebarnett could you explain to @saeedmb8 (or point him in the right direction?) what the meta tags with name & property attributes are for?

**Comment by smarzban** [comment]
Thanks @smn

**Comment by SteveBarnett** [comment]
@saeedmb8 Here's a quick rundown:

`<meta name="description" content="" />` is what appears in search engines results, [like this](https://cloud.githubusercontent.com/assets/1239497/9494078/3f261882-4c04-11e5-8a93-e6330cbc9a97.png). It's often filled from a "site descripton" or "page description" field in a CMS.

`<meta name="keywords" content="NDOH" />` was historically used by search engines, but these days the content of a page has more "weight." It's still useful to have these here. [The Foundation site](http://praekeltfoundation.org/, for example, has things like `consulting, mobile, web`.

`<meta name="viewport" content="width=device-width, initial-scale=1" />` is a nudge for mobile browsers to display the content properly for devices that have a hi-res screen (like many Androids these days, and most iPhones). It says something like "show the content at the width of the device." [Here's more on MDN](https://developer.mozilla.org/en/docs/Mozilla/Mobile/Viewport_meta_tag).

`<meta http-equiv="content-type" content="text/html;charset=utf-8" />` sets the content encoding. A shorter, nicer, version is `<meta charset=utf-8>`.

`<meta property="og:title" content="NDOH" />` and `<meta property="og:description" content="NDOH">` are [Open Graph](http://ogp.me/) things, basically both for "Facebook SEO". These can be the same as the regular `meta` `title` and `description`.

---

There's more of these kind of things in the excellent [documentation](https://github.com/h5bp/html5-boilerplate/blob/master/dist/doc/html.md#indexhtml) for [h5bp](https://html5boilerplate.com/).

---

How's that, @smn? :)

**Comment by smarzban** [comment]
Thanks @SteveBarnett for your wonderful explanation :) now they make sense to me :))

**Comment by SteveBarnett** [comment]
@saeedmb8 Welcome! \o/

**Comment by smarzban** [comment]
@smn I've removed the `NDOH` and left it empty until I get the contents for them.

**Comment by smarzban** [comment]
@smn @bruskiza @miltontony ready for review

**Comment by bruskiza** [comment]
:+1: even though they empty

**Pull request merged**